src/SerilogTracing/Instrumentation/HttpClient/HttpRequestOutActivityInstrumentor.cs

```diff
@@ -62,14 +62,21 @@ public void InstrumentActivity(Activity activity, string eventName, object event
                 {
                     return;
                 }
-                
-                activity.AddTag("StatusCode", response != null ? (int)response.StatusCode : null);
 
-                if (activity.Status == ActivityStatusCode.Unset &&
-                    _requestTaskStatusAccessor.TryGetValue(eventArgs, out var requestTaskStatus))
+                var statusCode = response != null ? (int?)response.StatusCode : null;
+                activity.AddTag("StatusCode", statusCode);
```

Comment:
This is existing code, but should probably be `ActivityInstrumentation.SetLogEventProperty`.